app/src/main/java/com/example/androidtraining/MainActivity.kt

```diff
@@ -37,10 +84,141 @@ fun Greeting(name: String, modifier: Modifier = Modifier) {
     )
 }
 
-@Preview(showBackground = true)
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun GreetingPreview() {
-    AndroidTrainingTheme {
-        Greeting("Android")
+fun HomeScreen(navigateToProductScreen: () -> Unit = {}) {
```

Comment:
This is actually the Login screen, we'll have a Home one soon, too. On another note, please extract the bigger views/screens in separate files, it's more readable and you'll have easier time working on those in the future.

Reply:
fixed